runners/direct-java/src/main/java/org/apache/beam/runners/direct/WatermarkManager.java

```diff
@@ -789,13 +844,19 @@ public Instant get() {
    */
```

Comment:
update javadoc `@param` for the new field.

Reply:
Done.